app/javascript/packages/document-capture/components/back-button.tsx

```diff
@@ -10,7 +10,8 @@ interface BackLinkProps extends ComponentProps<typeof Button> {
 function BackButton({ includeBorder = false, ...props }: BackLinkProps) {
   const button = (
     <Button isUnstyled {...props}>
-      &#x2039; {t('forms.buttons.back')}
+      <span aria-hidden="true">&#x2039; </span>
```

Comment:
As I said in slack, the screenreader read "back" twice because it read "back" for both the arrow "<" and the word "back"

I fixed this glitch by adding `aria-hidden="true"` to the arrow "<". Documentation notes that `aria-hidden` is a way to fix [duplicated content](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden#description).